app/models/article.rb

```diff
@@ -1,2 +1,17 @@
 class Article < ApplicationRecord
+  def even?
+    Time.current.to_i.even?
+  end
+
+  def odd?
+    Time.current.to_i.odd?
+  end
```

Comment:
⚠️ **[undercover]** <Untested instance method> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Instance method `odd?` is missing coverage for line 7 (node coverage: 0.0)
<!-- __reviewdog__:ChA1YjRkZGM0YzJmNGM1ZTRlEgp1bmRlcmNvdmVy -->